coalib/bearlib/aspects/root.py

```diff
@@ -113,6 +113,15 @@ class Root(aspectbase, metaclass=aspectclass):
       ...
     coalib.bearlib.aspects.taste.TasteError:
     Root.Formatting.Greatness.min_greatness is not available ...
+
+    Replace the dummy `Formatting` aspect made in the doctests
+    above, with the actual functional `Formatting` aspect. 
```

Comment:
This would go into our documentation which is confusing and may be unnecessary. 
Please try to add an extra doctest to the very bottom this file, not attached to a function, so it doesnt appear in our documentation.

Reply:
Okay I am trying, meanwhile could you please review the comment as well (to make things a bit fast :) ).